recipes-samples/images/rpb-console-image-lava.bb

```diff
@@ -0,0 +1,5 @@
+require rpb-console-image.bb
+
+CORE_IMAGE_BASE_INSTALL += " \
+    linaro-lava-tests \
```

Comment:
what is linaro-lava-tests? The only thing I find is linaro-lava-tests.bb in meta-linaro, but even with that , i don't see how it's doing what we are supposed to do..